test-apps/ui-test-app/src/frontend/appui/imodelopen/IModelOpen.tsx

```diff
@@ -67,14 +67,11 @@ export class IModelOpen extends React.Component<IModelOpenProps, IModelOpenState
       });
     }
 
-    if (undefined === this.props.getAccessToken)
-      return;
-
     const token = await this.props.getAccessToken();
-    if (undefined === token)

```

Comment:
@NancyMcCallB when I ran ui-test-app I got an error that the token was blank. I think this is why, but now I just get "fetching project information" spinner and i'm not sure how to debug.

Reply:
@NancyMcCallB is ui-test-app working correctly for you now?